gwt-event-logical-gwt2-tests/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.gwtproject.event</groupId>
     <artifactId>gwt-event-parent</artifactId>
-    <version>HEAD-SNAPSHOT</version>
+    <version>${revision}</version>
   </parent>
   <artifactId>gwt-event-logical-gwt2-tests</artifactId>
   <packaging>gwt-lib</packaging>
```

Comment:
gwt-lib should be removed, then we can also remove the moduleName from the gwt-maven-plugin

Reply:
My IDE does not like it, in case the gwt-maven-plugin has no moduleName tag.

Reply:
Sorry, I didn't test this, this is what I thought Thomas explained to us via email. Perhaps I misunderstood or misread, will check.